utility/make_private_namespace.py

```diff
@@ -0,0 +1,45 @@
+""" Create a namespace """
+import os
+import subprocess
+import shlex
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def make_private_namespace(name, region='us-east-2'):
```

Comment:
create_...

Reply:
Done.  I should probably change the name of the file to create_private_namespace but I am not sure if this would confuse the review process.